adapters/infoawarebidder_test.go

```diff
@@ -173,7 +173,7 @@ func TestImpFiltering(t *testing.T) {
 		}
 
 		// Extra MakeRequests() call check: our mockBidder returns an adapter request for every imp
-		assert.Len(t, actualAdapterRequests, test.expectedImpLen, "Test failed. Incorrect lenght of filtered imps: %s", test.description)
+		assert.Len(t, actualAdapterRequests, test.expectedImpLen, "Test failed. Incorrect length of filtered imps: %s", test.description)
```

Comment:
I spell `length` wrong all the time. 😄

Reply:
I apparently spell parameters wrong all the time. :) I noticed most of these typos are from me. Doh!